backend/src/core/features/game-identifier/game.identifier.js

```diff
@@ -84,6 +86,48 @@ export class GameIdentifier {
     await this.#steamAppsRepository.updateSteamAppsById(steamApps);
   }
 
+  checkIfGameViaSteamApi = async () => {
+    this.#logger.debugc(`identifying games via Steam API`);
+
+    const steamApps = await this.#steamAppsRepository.getSteamApiUntriedFilteredSteamApps(
+      this.#options.batchSize,
+    );
+
+    if (steamApps.isEmpty) {
+      this.#logger.debugc(
+        `no Steam API checkable steam apps in db, retry in: ${
+          this.#options.globalIterationDelay
+        } ms`,
+      );
+
+      return;
+    }
+
+    const steamApiApps = await this.#getSteamAppsViaSteamApi(steamApps.content);
```

Comment:
No need to provide the entire content, only the IDs are needed. Create a method and provide only the ids to the client.